ddl/index.go

```diff
@@ -669,6 +669,12 @@ func onDropIndex(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		// Set column index flag.
 		dropIndexColumnFlag(tblInfo, indexInfo)
 
+		failpoint.Inject("MockPanicInCancellingAddIndex", func(val failpoint.Value) {
```

Comment:
Could we use `mockExceedErrorLimit` instead of `MockPanicInCancellingAddIndex`? Then we can reduce a mock variable.